pkg/preparer/orchestrate.go

```diff
@@ -165,6 +169,27 @@ func (p *Preparer) handlePods(podChan <-chan pods.PodManifest, quit <-chan struc
 	}
 }
 
+// check if a manifest satisfies the signature requirement of this preparer
+func (p *Preparer) verifySignature(manifest pods.PodManifest, logger logging.Logger) bool {
+	// do not remove the logger argument, it's not the same as p.Logger
+	if p.keyring == nil {
```

Comment:
Per our discussion: let's remove the unauthenticated code path and provide some dummy keys that are used in development/testing environments. I'm happy to help with this.